docs/.nojekyll

```diff
@@ -0,0 +1 @@
+TypeDoc added this file to prevent GitHub Pages from using Jekyll. You can turn off this behavior by setting the `githubPages` option to false.
```

Comment:
is this necessary?

Reply:
hmmm, i think `yarn docs` is not correct we probably should add `typedoc --githubPages false` as a `yarn docs` package

Reply:
I don't know. It got generated by typedoc so if we don't need it we should not generate it by turning it off by that `githubPages` setting? Where are those settings?

Reply:
`typedoc --githubPages false` added ✅